firmware/src/system_config/number1/system_config.h

```diff
@@ -0,0 +1,196 @@
+   The PLIB function attribute that causes the warning.  Once that issue has been
+   resolved, this definition should be removed. */
+#define _PLIB_UNSUPPORTED
+
+
+// *****************************************************************************
+// *****************************************************************************
+// Section: Included Files
+// *****************************************************************************
+// *****************************************************************************
+/*  This section Includes other configuration headers necessary to completely
+    define this configuration.
+*/
+
+
+// *****************************************************************************
+// *****************************************************************************
+// Section: System Service Configuration
+// *****************************************************************************
+// *****************************************************************************
+
+
+// *****************************************************************************
+/* Clock System Service Configuration Options
+*/
+#define SYS_CLK_FREQ                        80000000ul
+#define SYS_CLK_BUS_PERIPHERAL_1            80000000ul
+#define SYS_CLK_UPLL_BEFORE_DIV2_FREQ       48000000ul
+#define SYS_CLK_CONFIG_PRIMARY_XTAL         8000000ul
+#define SYS_CLK_CONFIG_SECONDARY_XTAL       0ul
+   
+/*** Interrupt System Service Configuration ***/
+
+#define SYS_INT                     true
+
+
+/*** Ports System Service Configuration ***/
+
+#define SYS_PORT_AD1PCFG        ~0xffff
+#define SYS_PORT_CNPUE          0x0
+#define SYS_PORT_CNEN           0x0
+  
+
+
+
+#define SYS_PORT_D_TRIS         0xff8
+#define SYS_PORT_D_ODC          0x0
+
+
+
+
+
+
+ 
+
+/*** Console System Service Configuration DISABLED ***/
+
+#define SYS_CONSOLE_MESSAGE(message)
+#define SYS_CONSOLE_PRINT(fmt, ...)
+
+ 
+/*** Command Processor System Service Configuration DISABLED ***/
+
+#define SYS_CMD_MESSAGE(message)
+#define SYS_CMD_PRINT(fmt, ...)
+#define SYS_CMD_READY_TO_READ()
+
+
+
+
+// *****************************************************************************
+// *****************************************************************************
+// Section: Driver Configuration
+// *****************************************************************************
+// *****************************************************************************
+
+// *****************************************************************************
+// *****************************************************************************
+// Section: Middleware & Other Library Configuration
+// *****************************************************************************
+// *****************************************************************************
+
+/*** USB Driver Configuration ***/
+
+/* Enables Device Support */
+#define DRV_USB_DEVICE_SUPPORT      true
+
+/* Enables Device Support */

```

Comment:
This is all autogenerated. The less we change it the easier it is to merge in the harmony changes.